easybuild/easyconfigs/p/Primer3/Primer3-2.5.0-foss-2020a.eb

```diff
@@ -0,0 +1,17 @@
+name = 'Primer3'
+version = '2.5.0'
+
+homepage = 'https://primer3.org/'
+description = """Primer3 is a widely used program for designing PCR primers (PCR = 'Polymerase Chain Reaction').
+ PCR is an essential and ubiquitous tool in genetics and molecular biology.
+ Primer3 can also design hybridization probes and sequencing primers."""
+
+toolchain = {'name': 'foss', 'version': '2020a'}
```

Comment:
I cannot see that this uses a maths library or MPI, so we can move it to the `GCC` toolchain.